### ISSUE TITLE
Make Bridge configuration take a set of name/value pairs

### ISSUE THREAD

**Issue by roncain** [dotnet/wcf#185]
Rather than just the location of the resources folder, the Bridge POST to configure should take a set of name/value pairs.  And those name/value pairs should initially come from TestProperties, like this:

``` c#
        IEnumerable<string> propertyNames = TestProperties.PropertyNames;
        foreach(var name in propertyNames)
        {
            string value = TestProperties.GetProperty(name);
            ... put name/value into dictionary to pass to Bridge POST
        }
```
There is already a reserved property called TestProperties.BridgeResourceFolder_PropertyName, so the value used during the build will be passed to the Bridge, and the Bridge can pickup the resource folder name from the name/value pairs.

**Comment by roncain** [comment]
Issue #186 should be addressed before doing this issue so that the Bridge's resource folder correction in the set of name/value pairs is correct.

**Comment by roncain** [comment]
Resolved by #215
